install-deps.sh

```diff
@@ -231,6 +231,16 @@ EOF
     fi
 }
 
+for_make_check=false
+if tty -s; then
+    # interactive
+    for_make_check=true
+elif [ $FOR_MAKE_CHECK ]; then
+    for_make_check=true
+else
+    for_make_check=false
```

Comment:
Redundant? Either initially set or set in else clause.

Reply:
hmm, i wanted to "declare" `for_make_check` to help the readers to understand what value it might have.